Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h

```diff
@@ -31,6 +31,22 @@
   #define BOARD_CUSTOM_BUILD_FLAGS -DLPC_PINCFG_UART3_P4_28
 #endif
 
+//
+// EEPROM SD
+//
+//uncomment to enable EEPROM simulation on the SD card
+//#define SDCARD_EEPROM_EMULATION
+
+//
+//EEPROM I2C 
+//
+// uncomment to enable the EEPROM I2C module
+//#define I2C_EEPROM
+//#ifdef E2END
+// #undef E2END
+//#endif
+//#define E2END 0x7FFF // koncová adresa EEPROM 24C256 (32 kB = 8 kB)
```

Comment:
You should use `MARLIN_EEPROM_SIZE` instead.

Reply:
Yes, tested.
#define MARLIN_EEPROM_SIZE 0x8000 // 32Kb (24C256)
both variants are functional.
So you recommend to convert to MARLIN_EEPROM_SIZE?

Reply:
> So you recommend to convert to MARLIN_EEPROM_SIZE?

Yes